api/app/controllers/spree/api/v2/shipments_controller.rb

```diff
@@ -0,0 +1,155 @@
+module Spree
+  module Api
+    module V2
+      class ShipmentsController < Spree::Api::BaseController
+
+        before_action :find_and_update_shipment, only: [:ship, :ready, :add, :remove]
+        before_action :load_transfer_params, only: [:transfer_to_location, :transfer_to_shipment]
+
+        def mine
+          if current_api_user.persisted?
+            @shipments = Spree::Shipment
+              .reverse_chronological
+              .joins(:order)
+              .where(spree_orders: {user_id: current_api_user.id})
+              .includes(mine_includes)
+              .ransack(params[:q]).result.page(params[:page]).per(params[:per_page])
```

Comment:
Line is too long. [84/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.